src/model.js

```diff
@@ -771,6 +771,52 @@ ${associationOwner._getAssociationDebugList()}`);
     });
   }
 
+  /**
+   * Indexes created from options.indexes when calling Model.init
+   */
+  static _manualIndexes;
+
+  /**
+   * Indexes created from {@link ModelAttributeColumnOptions.unique}
+   */
+  static _attributeIndexes;
+
+  static getIndexes() {
+    return [
+      ...(this._manualIndexes ?? []),
+      ...(this._attributeIndexes ?? []),
+      ...(this.uniqueKeys ? Object.values(this.uniqueKeys) : []),
+    ];
+  }
+
+  static get _indexes() {
+    throw new Error('Model._indexes has been replaced with Model.getIndexes()');
```

Comment:
This will land in v7, right? => Can we note this under breaking changes? Should we deprecate this in a new v6 release?

Reply:
Yep, v7 only

It's an undocumented, untyped, weakly private API. I've added the error in case people were using it, and we can of course add it in our list of breaking changes but I don't know how to go about deprecating this in v6 as we use it internally and us accessing the property would print the deprecation warning

Reply:
Technically we could move it to a new function in v6, reference that one internally and add a deprecation message to _indexes right?

Reply:
We could, but then removing that function in V7 becomes the breaking change (or of we use `getIndexes` like in V7, then the breaking change is that it's behavior is not the same in V6 vs v7 because _indexes and getIndexes are not strictly equivalent)